### PR TITLE
Task #1032251 live validator swaggerPathPattern

### PR DESCRIPTION
**Pull request by raych1**
Change the swaggerPathPattern to array type from string.

### CODE REVIEW

File: lib/validators/liveValidator.ts [hunk 1]
```diff
@@ -781,6 +781,26 @@ export class LiveValidator {
     return op as OperationWithApiVersion
   }
 
+  private async getMatchedPaths(jsonsPattern: string): Promise<string[]> {
+    const matchedPaths = await globby(jsonsPattern, {
```
Comment (nickzhums) [inline lib/validators/liveValidator.ts]: is it possible to directly pass an array of strings to globby? if so maybe we can avoid concating the swagger paths

Comment (raych1) [reply lib/validators/liveValidator.ts]: Good suggestion, changed it.

File: lib/validators/liveValidator.ts [hunk 2]
```diff
@@ -36,7 +36,7 @@ export interface LiveValidatorOptions {
   }
   useRelativeSourceLocationUrl?: boolean
   directory: string
-  swaggerPathsPattern: string
+  swaggerPathsPattern: string[]
```
Comment (lirenhe) [inline lib/validators/liveValidator.ts]: Can this be string | string[]?

File: lib/validators/liveValidator.ts [hunk 3]
```diff
@@ -781,35 +781,48 @@ export class LiveValidator {
...
       )
       return this.options.swaggerPaths
     } else {
-      const allJsonsPattern = "/specification/**/*.json"
-      const jsonsPattern = path.join(
-        this.options.directory,
-        this.options.swaggerPathsPattern || allJsonsPattern
-      )
-      const swaggerPaths = await globby(jsonsPattern, {
-        ignore: [
```
Comment (lirenhe) [inline lib/validators/liveValidator.ts]: Why is this removed?

File: lib/validators/liveValidator.ts (hunk 2 above)
Comment (raych1) [reply lib/validators/liveValidator.ts]: It can but it's not allowed to pass by chart in validation service.
